tests/components/lametric/conftest.py

```diff
@@ -84,6 +84,24 @@ def mock_lametric() -> Generator[MagicMock, None, None]:
         yield lametric
 
 
+@pytest.fixture
+def mock_lametric_sky(request) -> Generator[MagicMock, None, None]:
+    """Return a mocked LaMetric SKY client."""
+    with patch(
```

Comment:
We do not have to create a new mock, it is the same thing, with a different json fixture loaded.

Just create a device fixture, for example:

https://github.com/home-assistant/core/blob/c1c319d4d1500bcc8054bddba17e16873832b5c3/tests/components/wled/conftest.py#L45-L65

Reply:
Thanks for the example, I've added a device fixture now :+1: